lib/record_store/provider/dynect.rb

```diff
@@ -83,15 +87,21 @@ def secrets
     end
 
     def build_from_api(api_record)
+      record_type = api_record.fetch('record_type')
       record = api_record.merge(api_record.fetch('rdata')).slice!('rdata').symbolize_keys
 
-      return if record.fetch(:record_type) == 'SOA'
+      return if record_type == 'SOA'
+
+      case record_type
+      when 'ALIAS'
+        record[:cname] = record.delete(:alias)
+      end
```

Comment:
@es Thanks for your note! I just realized, although this would require: 
- In record-store, change all DNSimple ALIAS records' `cname` attributes to `alias`, OR: 
- (Dirty, but minimal impact) Make DNSimple read the `cname` attribute if the `alias` attribute is empty here: https://github.com/Shopify/record_store/blob/master/lib/record_store/provider/dnsimple.rb#L138

what do you think?